src/scenes/Projects/List/ProjectList.tsx

```diff
@@ -60,22 +97,53 @@ export const ProjectList: FC = () => {
           </FilterButtonDialog>
         </Grid>
       </Grid>
-      <Typography variant="h3" paragraph>
-        {data ? (
-          `${formatNumber(data.projects.total)} Projects`
-        ) : (
-          <Skeleton width="12ch" />
-        )}
-      </Typography>
-      <ListContainer>
-        {listOrPlaceholders(data?.projects.items, 5).map((item, index) => (
-          <ProjectListItemCard
-            key={item?.id ?? index}
-            project={item}
-            className={classes.projectItem}
-          />
-        ))}
-      </ListContainer>
+      <TabContext value={activeTab}>
+        <Tabs
+          value={activeTab}
+          onChange={(_e, newActiveTab) => setActiveTab(newActiveTab)}
+          aria-label="project navigation tabs"
+        >
+          <Tab label="My Projects" key="mine" value="mine" />
+          <Tab label="All" key="all" value="all" />
+          <Tab label="Recent" key="recent" value="recent" />
+          <Tab label="Pinned" key="pinned" value="pinned" />
+        </Tabs>
+        <Typography variant="h3" className={classes.projectTotal}>
+          {data ? `${getProjectTotal()} Projects` : <Skeleton width="12ch" />}
+        </Typography>
+        <ListContainer>
+          <TabPanel value="mine">
+            {listOrPlaceholders(data?.projects.items, 5).map((item, index) => (
```

Comment:
Not sure about this one off the top of my head. But all tabs will come from the same list query, so I don't think there's a need to specify all of them. Or maybe any of them?